### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by panther7**
Improvements, see commits.

### CODE REVIEW

File: wingetui/uiSections.py [hunk 1]
```diff
@@ -1919,7 +1931,30 @@ def __init__(self, parent = None):
...
+            "ppvnf: Portuguese (Portugal and Brazil)",
+            "regulargvy13: Polish",
+            "Rosario Di Mauro: Italian",
+            "Satyam Singh Niranjan: Hindi",
+            "Sergey: Russian",
+            "sho9029: Japanese",
+            "TAKASE, Yuki: Japanese",
+            "yrctw: Chinese",
+            "Артем: Russian",
+            ):
```
Comment (panther7) [inline wingetui/uiSections.py:1958]: TODO: Generate from lang files

Comment (marticliment) [reply wingetui/uiSections.py:1958]: I don't think that can be done, at least without the Tolgee api...

Comment (panther7) [reply wingetui/uiSections.py:1958]: Via translation key, as "Translated to English by martinet101" in ElevenClock

Comment (marticliment) [reply wingetui/uiSections.py:1958]: can you please open an issue regarding this?

Thanks!

**Pull request merged**